activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -48,8 +48,8 @@ def clear(options = nil)
             @data.clear
           end
 
-          def fetch(*args, &block)
-            @data.fetch(*args, &block)
+          def fetch_entry(key, options=nil) # :nodoc:
```

Comment:
Not sure if we can just yank out `fetch`, since it didn't have `:nodoc:` it's part of the public API and people might be relying on it.

There should be spaces around `=` in `options = nil`.